### PR TITLE
DB refactor

### PR DESCRIPTION
**Pull request by BrokenShell**
Adds new table for DS to store case data.

### CODE REVIEW

**Comment by BrokenShell** [comment]
This is the currently deployed branch. v. 0.35.6

File: app/main.py [hunk 1]
```diff
@@ -57,90 +49,10 @@ async def pdf_ocr(uuid: str):
...
-    
-    conn = psycopg2.connect(db, sslmode="require")
-    curs = conn.cursor()
-
-    df = pd.read_sql(query,conn)
-    conn.close()
-    return get_stacked_bar_chart(df, column_to_graph)
-
-
-@app.post("/vis/judges/")
```
Comment (Dylan-Sivori) [inline app/main.py:87]: Were these endpoints meant to be deleted? We would be getting rid of the visualization endpoints, is that correct?

**Review by Frank-Howd — APPROVED** [review]
These changes look great; as always, amazing work, Robert.

**Review by Dylan-Sivori — COMMENTED** [review]
The db_ops file seems like a very smart way to manage handling database operations. On main.py it looks like our visualization endpoints are being removed, is that what you are looking to do in this merge?

File: app/main.py (hunk 1 above)
Comment (BrokenShell) [reply app/main.py:87]: Yes, the visualization endpoints are under development.

Comment (BrokenShell) [reply app/main.py:87]: Don't worry, they'll be back

Comment (Dylan-Sivori) [reply app/main.py:87]: Understood, thank you for letting us know!

**Review by Dylan-Sivori — APPROVED** [review]

**Pull request merged**